docs/Model_Assumptions.rst

```diff
@@ -603,6 +603,13 @@ The perfect foresight can lead to suspicious dispatch of assets, for example cha
 
 .. _limitations-missing-kpi:
 
+
+Optimization precision
+######################
+
+The MVS makes use of the open energy modelling framework (oemof) by using oemof-solph. It is defined that the energy system is solved with the `cbc-solver` and a `ratioGap=0.03`. This influences the precision of the optimized decision variables, ie. the optimized capacities as well as the dispatch of the assets.
```

Comment:
Isn't it possible to use another solver? Here it sounds like one would not be able to use another solver.